projects/pages/src/public-api.ts

```diff
@@ -1,5 +1,9 @@
 export * from './modules/action-hub/action-hub.module';
+export * from './modules/action-hub/action-hub.component';
+export * from './modules/action-hub/action-hub-buttons.component';
+export * from './modules/action-hub/action-hub-content.component';
 export * from './modules/action-hub/types/action-hub-needs-attention';
 export * from './modules/action-hub/types/page-link';
 export * from './modules/action-hub/types/recent-link';
 export * from './modules/page-header/page-header.module';
+export * from './modules/page-header/page-header.component';
```

Comment:
Components should be exported using the lambda character, for example:
```
export { SkyPageHeaderComponent as λ1 } from './modules/page-header/page-header.component';
```

Reply:
See: https://github.com/blackbaud/skyux-core/blob/5.0.0-next/projects/core/src/public-api.ts#L68-L72